From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ian Wahbe <me@iwahbe.com>
Date: Thu, 12 Sep 2024 20:06:09 +0200
Subject: [PATCH] Work around
 https://github.com/pulumi/pulumi-terraform-bridge/issues/1667


diff --git a/github/migrate_github_branch_protection.go b/github/migrate_github_branch_protection.go
index 2eccae49..42dcc1f2 100644
--- a/github/migrate_github_branch_protection.go
+++ b/github/migrate_github_branch_protection.go
@@ -64,7 +64,9 @@ func resourceGithubBranchProtectionUpgradeV1(_ context.Context, rawState map[str
 	var blocksCreations bool = false
 
 	if v, ok := rawState["blocks_creations"]; ok {
-		blocksCreations = v.(bool)
+		// Equate type errors as false. This is a work-around for
+		// https://github.com/pulumi/pulumi-terraform-bridge/issues/1667.
+		blocksCreations, _ = v.(bool)
 	}
 
 	if v, ok := rawState["push_restrictions"]; ok {
